otwcode/otwarchive
# AO3-6149 Redirect to 404 on comments index page as an admin

# Pull Request Checklist

  as the *first* thing in your pull request title (e.g. `AO3-1234 Fix thing`)
  until they are reviewed and merged before creating new pull requests.

## Issue

https://otwarchive.atlassian.net/browse/AO3-6149 

## Purpose

What does this PR do?

This PR removes the functionality to show the most recent comments in A03 when logged in as an admin (at the `/comments` URL).

## Testing Instructions

How can the Archive's QA team verify that this is working as you intended?

You can test this is working by doing the following: 
- Go to http://test.archiveofourown.org/comments
  - This page should return 404 when logged in as admin and when logged in as a normal user
- You can view the comments index on works, tags, and admin posts as usual.

## Credit

I've created a jira account as well!

Credit: Chantal, she/her
